### ISSUE TITLE
Server: Activity Stream

### ISSUE THREAD

**Issue by cristi8** [specklesystems/speckle-server#293]
Details in the dedicated notion page.

- [x] Services
- [x] API
- [ ] Webhook system connection (depends on #294 )

**Comment by cristi8** [comment]
Closing this, as the Webhooks are tracked in a separate ticket (and in order to have webhooks functional, we have to connect them to activity streams anyway)
